EventListener/TemplateListener.php

```diff
@@ -88,16 +94,6 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
         $owner = $template->getOwner();
         list($controller, $action) = $owner;
 
-        // when no template has been given, try to resolve it based on the controller
-        if (null === $template->getTemplate()) {
-            if ($action === '__invoke') {
```

Comment:
This is not valid anymore since https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/398.